Libplanet.MySQLStore/Libplanet.MySQLStore.csproj

```
@@ -0,0 +1,64 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>Libplanet.MySqlStore</PackageId>
+    <Title>Libplanet.MySqlStore</Title>
+    <Summary>A Libplanet.IStore implementation using MySql</Summary>
+    <Description>A Libplanet.IStore implementation using MySql</Description>
```

How about using the official name *MySQL*? https://www.mysql.com/

```suggestion
    <Summary>A Libplanet.IStore implementation using MySQL</Summary>
    <Description>A Libplanet.IStore implementation using MySQL</Description>
```